mordred/task_manager.py

```diff
@@ -84,7 +86,9 @@ def run(self):
                 time.sleep(self.timer)
 
             for task in self.tasks:
-                task.run()
+                try:
+                    task.run()
```

Comment:
Ups, you can not start threads in this way. You must use task.start() which really create a new thread and then execute "run" with it. If you call directly to run, you are not creating a new thread, but just executing the run method in the thread object. This is why you can catch the Exception in this point.

If you use "start" a new thread is started and, if there are exceptions in the thread, you won't catch them here. You need to catch the exception inside the thread and put it in the queue, but inside the thread.